icu4j/main/core/src/main/java/com/ibm/icu/number/FormattedNumber.java

```diff
@@ -142,7 +142,7 @@ public MeasureUnit getOutputUnit() {
      * supported yet.
      *
      * @return NounClass
-     * @draft ICU 71.
+     * @stable ICU 71
```

Comment:
```suggestion
     * @stable ICU 72
```
See team email:

I think we should change the `FormattedNumber.getNounClass()` to `@stable ICU 72` (not 71) because in 72, when the new `DisplayOptions` were added, this method changed from returning a `com.ibm.icu.util.NounClass` to returning a `com.ibm.icu.text.DisplayOptions.NounClass`.
And, it seems confusing that the type it returns is newer than the method returning that type.